actions/sequester/Quest2GitHub/QuestGitHubService.cs

```diff
@@ -8,60 +8,49 @@ namespace Quest2GitHub;
 /// GitHub issues with Quest work items (typically User Stories).
 /// </summary>
 /// <remarks>
-/// Client applications should create an insteance of this class at startup.
+/// Client applications should create an instance of this class at startup.
 /// </remarks>
-public class QuestGitHubService : IDisposable
+/// <remarks>
+/// Initialize the service.
+/// </remarks>
+/// <param name="client">GitHub client</param>
+/// <param name="ospoKey">MS Open Source Programs Office personal access token</param>
+/// <param name="azdoKey">Azure Dev Ops personal access token</param>
+/// <param name="questOrg">The Azure Dev ops organization</param>
+/// <param name="questProject">The Azure Dev ops project</param>
+/// <param name="areaPath">The area path for work items from this repo</param>
+/// <param name="importTriggerLabel">The text of the label that triggers an import</param>
+/// <param name="importedLabel">The text of the label that indicates an issue has been imported</param>
+/// <param name="bulkImport">True if this run is doing a bulk import.</param>
+/// <remarks>
+/// The OAuth token takes precedence over the GitHub token, if both are 
+/// present.
+/// </remarks>
+public class QuestGitHubService(
+    IGitHubClient client,
+    string ospoKey,
+    string azdoKey,
+    string questOrg,
+    string questProject,
+    string areaPath,
+    string importTriggerLabel,
+    string importedLabel,
+    bool bulkImport) : IDisposable
 {
     private const string LinkedWorkItemComment = "Associated WorkItem - ";
-
-    private readonly IGitHubClient _ghClient;
-    private readonly QuestClient _azdoClient;
-    private readonly OspoClient _ospoClient;
-    private readonly string _areaPath;
-    private readonly string _questLinkString;
-    private readonly string _importTriggerLabelText;
-    private readonly string _importedLabelText;
+    private const string PreviewBuildComment = "<!-- PREVIEW-TABLE-START -->";
+    private readonly IGitHubClient _ghClient = client;
+    private readonly QuestClient _azdoClient = new(azdoKey, questOrg, questProject);
+    private readonly OspoClient _ospoClient = new(ospoKey, bulkImport);
+    private readonly string _areaPath = areaPath;
+    private readonly string _questLinkString = $"https://dev.azure.com/{questOrg}/{questProject}/_workitems/edit/";
+    private readonly string _importTriggerLabelText = importTriggerLabel;
+    private readonly string _importedLabelText = importedLabel;
```

Comment:
Replace `_importTriggerLabelText` with `importTriggerLabel` and `_importedLabelText` with `importedLabel` in this file, and delete these lines:

```suggestion
```